### ISSUE TITLE
Feature request: Enable access log

### ISSUE THREAD

**Issue by nockstarr** [Hellowlol/HTPC-Manager#485]
Would be nice to be able to enable/disable(setting) access logging for the web GUI.

Even nicer if access logs would be logged to a separate file in sted of flooding the "htpcmanager.log" file.

I know i can go in "log.py" to enable access logging manually,  but that is flooding the htpcmanager log and makes it useless(my opinion).

If this is highly unwanted, I would like to know how to enable access logging to a separate file manually if possible:)

Please and Thank you:)

**Comment by Hellowlol** [comment]
Access log like logins etc? Or are you refering to cherrpy access log? Remember that you can filter out stuff easily

**Comment by nockstarr** [comment]
Yes I was talking about cherrypy access log. What do you mean filter out? Choose "info", "waring" etc.. from log in web gui?

**Comment by nockstarr** [comment]
I see that the filter is working nice. However make a setting for enableling cherrypy access log would be awsm:) Forget the separate file thingi.
